### DISCUSSION TITLE
Have a menu option to save the application file and ui file in ready-to-go format

### DISCUSSION THREAD

**Discussion by gcanyon** [alejandroautalan/pygubu-designer#218] [Ideas]
Going through the read me, while it's simple enough to copy-paste the application text and then make the appropriate substitutions, it would also be trivial to write that out at the same time as outputting the .ui file, right? And coding it once in the designer app is more efficient than having developers using the designer have to do it every single time they create an application.

**Comment by alejandroautalan** [comment]
Hello @gcanyon, thanks for trying pygubu.

I'm working on several issues that will improve code generation. One of them is to store the code generation options in the UI file.
I will add your requirement as additional option for consideration.

In the meantime, please read the following recommendations to avoid having to rewrite the code entered manually:

* https://github.com/alejandroautalan/pygubu-designer/issues/99
* [wiki/Design-Code-Iteration](https://github.com/alejandroautalan/pygubu-designer/wiki/Design-Code-Iteration)

If you have any other comment, please let me know.


Regards
Alejandro A.

**Comment by gcanyon** [comment]
Thanks, that’s very helpful, especially the wiki page!

FYI, I’m just starting out with Python. It’s been on my to-do list for
literally over ten years, but I kept putting it off because the tool I use
— LiveCode — has a tightly integrated GUI toolkit. And by “tightly” I mean
it’s live and interactive: you drag a button into a window, open its
script, add code, close the script, click the button, and the script
executes.

So as you can imagine, I’m always looking for the shortest, easiest, mostt
efficient path from “I have an idea!” to “Here’s my app with an interface.”

Thanks for Pygubu!

Regards,

Geoff
